web/packages/teleport/src/Discover/SelectResource/SelectResource.tsx

```diff
@@ -608,22 +623,43 @@ const Grid = styled.div`
   row-gap: 15px;
 `;
 
-const ResourceCard = styled.div<{ hasAccess?: boolean }>`
+const NewTabIconWrapper = styled.div`
   display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  justify-content: flex-end;
+  position: absolute;
+  top: ${props => props.theme.space[3]}px;
+  right: ${props => props.theme.space[3]}px;
+  opacity: 1;
+`;
+
+const ResourceCard = styled.button<{ hasAccess?: boolean }>`
   position: relative;
-  align-items: center;
+  text-align: left;
   background: ${props => props.theme.colors.spotBackground[0]};
   transition: all 0.3s;
 
+  border: none;
   border-radius: 8px;
   padding: 12px 12px 12px 12px;
+  padding-block-start: 12px;
+  padding-block-end: 12px;
```

Comment:
We already have `padding` in the line above, is it a testing leftover?

Reply:
Yes, I'll try removing this as well though - also added this after changing Div to a Button